### DISCUSSION TITLE
It is not possible in VEAT forms (EF25, EF26, EF27, EF28) to publish a notice that only contains lots that are not award.

### DISCUSSION THREAD

**Discussion by kletersteegp** [OP-TED/eForms-SDK#838] [Q&A] [answered]
It is not possible in VEAT forms (EF25, EF26, EF27, EF28) to publish a notice that only contains lots that are not award.
There is a rule in this forms that there should always be a settled contract. But is a lot is not awarded then there is no contract ... And our users want because of transparancy that a lot is not awarded. Is it possible to make this possible?

	<rule context="/*/ext:UBLExtensions/ext:UBLExtension/ext:ExtensionContent/efext:EformsExtension/efac:NoticeResult[$noticeSubType = '25']">
		<assert id="ND-SettledContract-25" role="ERROR" diagnostics="ND-SettledContract" test="count(efac:SettledContract) &gt; 0">rule|text|ND-SettledContract-25</assert>
	</rule>

**Comment by YvesJo** [comment, marked as answer]
Hi,
My understanding of DAP (Direct Award Preannouncement) is that these forms are used for directly awarded lots (Negotiated Procedure without prior Call for Competition). The lots in the notice are therefore awarded and BT-142 is useless. Following the award and the negotiation there is an intent to sign the contract in the future.
When finally the contract does not get signed, this may be reported in a Result notice.
May be the case you are referring to, is the one where a lot is directly awarded, the negotiations fail for that specific lot and it is already known at VEAT publication time that no contract will be signed for that specific lot. Should that lot then be in the notice? Should that form evolve so that directly awarded lots may not lead to a contract? This may require an Annex update and I recommend you to get direct in touch with DG GROW regarding that specific question.
KR
